packages/common/src/feedback/_internal/FeedbackSchema.ts

```diff
@@ -0,0 +1,23 @@
+import { IConfigurationSchema } from "../../core";
+import { HubItemEntitySchema } from "../../core/schemas/shared/HubItemEntitySchema";
+
+export type FeedbackEditorType = (typeof FeedbackEditorTypes)[number];
+export const FeedbackEditorTypes = [
+  "hub:feedback:edit",
+  "hub:feedback:settings",
+] as const;
+
+/**
+ * defines the JSON schema for a Feedback entity's editable fields
+ */
+export const FeedbackSchema: IConfigurationSchema = {
+  ...HubItemEntitySchema,
+  properties: {
+    displayMap: {
+      type: "boolean",
+      enum: [true, false],
+      default: false,
+    },
```

Comment:
We may need to add the other property `hasMapQuestion` to the schema for the UISchema to make the determination if the `Map display` section renders per [figma](https://www.figma.com/file/1bvSoMV4ygNqHGXmkIBgJd/Survey-workspace?type=design&node-id=8-2102&mode=design&t=lJwoAoQJBO6OR96C-0)